benchmark/camelyon/workflows.py

```diff
@@ -110,7 +111,7 @@ def substrafl_fed_avg(
         experiment_folder=Path(__file__).resolve().parent / "benchmark_cl_experiment_folder",
     )
 
-    clients[0].wait_compute_plan(compute_plan.key)
+    clients[0].wait_compute_plan(key=compute_plan.key, raise_on_failure=True)
```

Comment:
I'm wondering if while we're at fixing this test, we want to check anything else... 
Execution time has more or less an upper bound due to the timeout allocation, but maybe we should check that whatever performance metrics are returned are between 0 and 1. Maybe we can have a narrower interval given that the seed is fixed.
(Perfectly happy with postponing that to a follow-up issue since the priority now is to get the CI green again)

Reply:
Indeed, I was wondering what kind of additional controls could be applied here. This is a great idea. As it will take some time to do it right (ie monitoring metrics results on at least 2 or 3 runs) and there is a urge to fix the CI ; I have created a [follow-up issue](https://linear.app/owkin/issue/FL-1095/check-performance-result-of-camelyon-benchmark).